### ISSUE TITLE
iihs_sources_v3: All sources of economic times may not be configured, hence it's showing up missed article

### ISSUE THREAD

**Issue by salus-sage** [janastu/IIHS-TNUSSP-feed#16]
Valid rss urls can be found by view source in each category
Test case: i'm configuring politics and nation to test it out
http://economictimes.indiatimes.com/news/politics-and-nation/rssfeeds/1052732854.cms
The key pattern missing in config is /category/sub-cat/rssfeeds/id.cms

**Comment by khushpreet-kaur** [comment]
Structure added to the sheet.

**Comment by salus-sage** [comment]
These are sources with RSS, why shud they go on the sheet?

**Comment by khushpreet-kaur** [comment]
@salus-sage  True. Economics Times provide ~156 link on this page: http://economictimes.indiatimes.com/rss.cms 

Dealing with huge url list, I would rather prefer script than manually crawling(to verify, manually it took ~2.15hr in just crawling/copy-pasting/formatting link in sources_v3 for ET's 156 url, Which I feel is not required anymore!)

On top of that, my understanding is that if we have a script doing all the crawling/generating, why not update the script instead sources directly.  This is anyway more reliable method, individual pasted url may change in future and sources file may get stale.  

What do you think?

**Comment by salus-sage** [comment]
But the crawlers are to generate RSS. Not to just scrape links. For that u
remember V used a online service? I don't want more links in crawler list
until we crack the regex.

**Comment by salus-sage** [comment]
totally about 149 categories available, but configuration is partial on newsrack. can expect this source to show up in the missed articles category during testing
